src/rdkafka.h

```diff
@@ -4880,14 +4943,153 @@ struct rd_kafka_group_list {
  *           group list.
  *
  * @sa Use rd_kafka_group_list_destroy() to release list memory.
+ *
+ * @deprecated Use rd_kafka_describe_consumer_groups() instead.
  */
 RD_EXPORT
+RD_DEPRECATED
 rd_kafka_resp_err_t
 rd_kafka_list_groups(rd_kafka_t *rk,
                      const char *group,
                      const struct rd_kafka_group_list **grplistp,
                      int timeout_ms);
 
+/**
+ * @brief Creates an options struct for rd_kafka_list_consumer_groups.
+ *
+ * \p request_timeout the request timeout.
```

Comment:
Use `@param ..` when list parameters like this. `\p ..` is for use in sentences.

Reply:
dito